src/backend/main.py

```diff
@@ -164,22 +166,38 @@ def get_mirror_list(
     
     # protocol get arg
     request_protocol = request.args.get('protocol')
-    if request_protocol and request_protocol not in ["http","https"]:
-        return "Invalid input for protocol, valid options: http, https"
+    if request_protocol and request_protocol not in ['http', 'https']:
```

Comment:
Wouldn't 
```suggestion
    if request_protocol not in ['http', 'https']:
```
do the same?

Reply:
No. Because we return any mirrors if request_protocol is None